lib/generators/surveyor/install_generator.rb

```diff
@@ -62,10 +62,6 @@ def migrations
       end
     end
 
-    def routes
-      route('mount Surveyor::Engine => "/surveys", :as => "surveyor"')
-    end
-
     def assets
```

Comment:
Can we delete this too? We can also likely delete the locales generator as well.